.github/workflows/main.yml

```diff
@@ -161,6 +161,6 @@ jobs:
           ocaml-compiler: ${{ matrix.ocaml-compiler }}
           dune-cache: true
 
-      - run: opam depext ocamlformat=$(cat .ocamlformat | grep version | cut -d '=' -f 2) --install
+      - run: opam depext ocamlformat=$(cat .ocamlformat | grep version | cut -d '=' -f 2 | cut -d ' ' -f 1) --install
```

Comment:
```suggestion
      - run: opam depext --install ocamlformat=$(awk -F = '/version/{print $2}' .ocamlformat)
```

Reply:
How about simply moving the comment to a line above and replacing the step with this?

Reply:
There is not much difference, but I'm avoiding UUOC these days.
https://en.wikipedia.org/wiki/Cat_(Unix)#Useless_use_of_cat

Reply:
I'm not familiar with Unix commands in general so I fully trust you on that 😄 Thanks for the suggestion :)